src/runtime/lean.h

```diff
@@ -951,6 +951,15 @@ static inline lean_obj_res lean_byte_array_size(b_lean_obj_arg a) {
     return lean_box(lean_sarray_size(a));
 }
 
+/* static inline uint8_t lean_byte_array_uget(b_lean_obj_arg a, size_t i) { */
+/*     assert(i < lean_sarray_size(a)); */
+/*     return lean_to_sarray(a)->m_data[i]; */
+/* } */
+
+/* static inline uint8_t lean_byte_array_fget(b_lean_obj_arg a, b_lean_obj_arg i) { */
+/*     return lean_byte_array_uget(a, lean_unbox(i)); */
+/* } */
+
```

Comment:
Why are you adding these commented?

Reply:
I needed them and I forget to delete them. I'll delete them